cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -187,6 +188,8 @@ public void create(HttpRequest request, HttpResponder responder,
       }
     }
 
+    streamCoordinator.affectLeader(stream);
```

Comment:
In fact it should be called inside `StreamAdmin` as a Stream can be created either through REST API or from app-fabric (on app deployment). `StreamAdmin` is the center piece for the actual creation.